.github/workflows/create-rl-image.yml

```
@@ -0,0 +1,45 @@
+name: Build Rocky 9 image (docker)
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Rocky 9 image
+    runs-on:
+      - self-hosted
+      - packer
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run build
+        run: |
+          sh -vx
+          docker stop packer-rocky9 || true
+          docker rm packer-rocky9 || true
+
+          gid=$(getent group kvm | awk -F : '{ print $3 }')
+          sed -e "s/##KVM_GID##/$gid/" docker/Dockerfile.orig >Dockerfile
+          cat Dockerfile
+
+          docker build . \
+            --build-arg http_proxy \
+            --build-arg https_proxy \
+            --build-arg no_proxy \
+            --tag dataved/packer-rocky9
```

Let's use `infractl/packer-rocky9` instead?
```suggestion
            --tag dataved/packer-rocky9
```

---

Fixed